mri/test/test_wavelet_adjoint.py

```diff
@@ -80,6 +80,23 @@ def test_Wavelet3D_PyWt(self):
             numpy.testing.assert_allclose(x_d, x_ad, rtol=1e-6)
         print(" Wavelet3 adjoint test passes")
 
+    def test_Wavelet_UD_2D(self):
+        """Test the adjoint operation for Undecimated wavelet
+        """
+        for i in range(self.max_iter):
+            print("Process Wavelet Undecimated test '{0}'...", i)
+            wavelet_op_adj = WaveletUD2(nb_scale=4)
```

Comment:
why adj?

Reply:
Copied code from earlier test where it made sense. Fixing